en/guide/telemetry.md

```diff
@@ -182,43 +184,43 @@ This same approach can be used to report only messages that meet some condition.
 
 ## Blocking Telemetry Requests (State Management)
 
-Some commands/vehicle operations must be performed in a certain sequence, or can only be called when the vehicle reaches a certain state. 
-For example, in order to takeoff the vehicle must first be armed, and in order to arm, the vehicle must have a home position/GPS lock. 
-For these cases you will want to monitor for the vehicle being in a certain state and block other commands until it is ready to proceed. 
+Some commands/vehicle operations must be performed in a certain sequence, or can only be called when the vehicle reaches a certain state.
+For example, in order to takeoff the vehicle must first be armed, and in order to arm, the vehicle must have a home position/GPS lock.
+For these cases you will want to monitor for the vehicle being in a certain state and block other commands until it is ready to proceed.
 
-Often the easiest approach is to use synchronous methods and poll for the result. 
+Often the easiest approach is to use synchronous methods and poll for the result.
 For example, the following code shows how you might poll for whether the vehicle is ready to arm ([health_all_ok()](../api_reference/classmavsdk_1_1_telemetry.md#classmavsdk_1_1_telemetry_1ad6d833741b5576f07204d268c5cd4d06)).
 
 ```cpp
 // Check if vehicle is ready to arm
-while (!telemetry->health_all_ok()) {
-    std::cout << "Vehicle not ready to arm" << std::endl;
+while (!telemetry.health_all_ok()) {
+    std::cout << "Vehicle not ready to arm" << '\n';
     std::this_thread::sleep_for(std::chrono::seconds(1));
 }
 // now ready to arm ...
 ```
 
-Similarly, you can use the asynchronous method and block 
+Similarly, you can use the asynchronous method and block
 
 ```cpp
 {
-    std::cout << "Waiting for system to be ready" << std::endl;
+    std::cout << "Waiting for system to be ready" << '\n';
     auto prom = std::make_shared<std::promise<void>>();
     auto future_result = prom->get_future();
-    telemetry->health_all_ok_async(
+    telemetry.subscribe_health_all_ok(
     [prom](bool result) {
         //fulfill promise if health is OK
         if (result) {// health OK
             prom->set_value();
         }
     });
-    
+
     future_result.get(); //Block until promise is fulfilled.
     // Now ready to arm
 }
 ```
 > **Note** Note that `prom->set_value()` must only be called once.
-  If your code can potentially call this multiple times, remember to unsubscribe from the callback after it succeeds (e.g. using `telemetry->health_all_ok_async(nullptr)`).
+  If your code can potentially call this multiple times, remember to unsubscribe from the callback after it succeeds (e.g. using `telemetry.subscribe_all_ok_async(nullptr)`).

```

Comment:
Just checking, is it really `subscribe_all_ok_async` - ie not having "health" seems mismatch with `subscribe_health_all_ok`

Reply:
Oops it's `subscribe_health_all_ok`, thanks for finding that.

Reply:
No worries. I presume you'll tidy as a post-process.

Reply:
Done https://github.com/mavlink/MAVSDK-docs/pull/225.